src/MarkersHandler.js

```diff
@@ -90,12 +91,16 @@ var markersHandler = function (vjs) {
 	        markersMap = {},
 	        markersList = [],
 	        // list of markers sorted by time
-	    		currentMarkerIndex = NULL_INDEX,
+			currentMarkerIndex = NULL_INDEX,
 	        markerTip = null,
 	        breakOverlay = null,
 	        overlayIndex = NULL_INDEX;
 
-	    function sortMarkersList() {
```

Comment:
It is a little unclear how the publisher-specific options are being applied to the markers - maybe we need some comment here?

Reply:
I added a comment to clarify. Please see the new commit below this comment.